release/ray_release/cluster_manager/minimal.py

```diff
@@ -288,4 +288,5 @@ def terminate_cluster(self):
         pass
 
     def get_cluster_address(self) -> str:
-        return f"anyscale://{self.project_name}/{self.cluster_name}"
+        # return f"anyscale://{self.project_name}/{self.cluster_name}"
+        return f"anyscale://{self.cluster_name}"
```

Comment:
Can we keep the full cluster address?

Reply:
Ah I was testing it (i cannot run locally without it) I will revert it